client/verta/tests/test_endpoint/test_kafka.py

```diff
@@ -12,55 +12,61 @@
 class TestKafkaSettings:
 
     @hypothesis.given(
+        cluster_config_id=st.text(min_size=1),
```

Comment:
Note that the tests currently fail with `500 Server Error: Kafka config ID does not correlate with valid kafka configuration for url: https://hm.dev.verta.ai/api/v1/deployment/stages/2/kafka`, which checks out considering we don't have actually kafka configuration IDs to provide to dep-api.

Reply:
`cluster_config_id=st.text(min_size=1)` might not end up working out for this test 🤔 

We were able to use `st.text(min_size=1)` for the topic names because there was no need for them to be real values in the old Kafka functionality; this test could get away with sending garbage values.

Now that this needs to be tied to a real, existing, backend-saved Kafka config, we might need to overhaul this test to create one in the backend